Marlin/src/pins/stm32h7/pins_FYSETC_SPIDER_KING723.h

```diff
@@ -0,0 +1,417 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2024 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+// zzcatvs 2024-05-24
+
+#pragma once
+
+#include "env_validate.h"
+
+#if HOTENDS > 5 || E_STEPPERS > 5
+  #error "FYSETC SPIDER KING V1.1 supports up to 5 hotends / E-steppers."
+#endif
+
+#ifndef BOARD_INFO_NAME
+  #define BOARD_INFO_NAME "FYSETC SPIDER KING V1.1"
+#endif
+#ifndef DEFAULT_MACHINE_NAME
+  #define DEFAULT_MACHINE_NAME BOARD_INFO_NAME
+#endif
+
+#if ANY(NO_EEPROM_SELECTED, FLASH_EEPROM_EMULATION)
+  #undef NO_EEPROM_SELECTED
+  #ifndef FLASH_EEPROM_EMULATION
+    #define FLASH_EEPROM_EMULATION
+  #endif
+  #define EEPROM_PAGE_SIZE      (0x800UL) // 2K
+  #define EEPROM_START_ADDRESS  (0x8000000UL + (STM32_FLASH_SIZE) * 1024UL - (EEPROM_PAGE_SIZE) * 2UL)
+  #define MARLIN_EEPROM_SIZE    EEPROM_PAGE_SIZE
+#endif
+
+//
+// BLTouch and Z Probe
+//
+#if ANY(BLTOUCH, FIX_MOUNTED_PROBE)
+  #if ENABLED(Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN)
+    #define Z_MIN_PIN                       PA0   // PD0
+  #elif !defined(Z_MIN_PROBE_PIN)
+    #define Z_MIN_PROBE_PIN                 PA0   // PD0
+  #endif
+#endif
+
+//
+// Servos
+//
+#define SERVO0_PIN                          PA1   // PD1
```

Comment:
Do you know why this (and other pins) have comments of other pins? I can't find any information on this variant of the Spider King to confirm what the silkscreen labeling says.

Reply:
You can check here for more information: https://wiki.fysetc.com/FYSETC-SPIDER-KING
The pin definition is exactly the same, and the new STM32H723 chip was used after replacing the motherboard.

Reply:
Ahh. I didn't realize it was mostly the same as [`pins_FYSETC_SPIDER_KING407.h`](https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.1.x/Marlin/src/pins/stm32f4/pins_FYSETC_SPIDER_KING407.h).

This can be simplified a lot further & combined, but I'll have to do that once I'm back at a computer.